dropwizard-metrics-datadog/pom.xml

```diff
@@ -2,8 +2,8 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <parent>
         <artifactId>metrics-datadog-parent</artifactId>
-        <groupId>org.coursera</groupId>
-        <version>2.0.0-RC1</version>
+        <groupId>com.Surely-Health</groupId>
```

Comment:
it complained if i left the package name in another org and pushed to our org in github package repo